Makefile

```diff
@@ -0,0 +1,24 @@
+
+QUIPUCORDS_UI_CONTAINER_TAG ?= quipucords-ui
+
+ifndef DOCKER_HOST
+	PODMAN_SOCKET := $(shell podman machine inspect --format '{{.ConnectionInfo.PodmanSocket.Path}}' 2> /dev/null || podman info --format '{{.Host.RemoteSocket.Path}}' 2> /dev/null || echo -n "")
```

Comment:
can we make this block exclusive to macos?

On linux, `podman machine` does nothing, and setting DOCKER_HOST is not required.

Reply:
huh, OTOH quipucords has the same thing 🤔 

I wonder if DOCKER_HOST is there just for the integration test that calls docker api (and will be removed soon).

Regardless, I'll leave this one approved. We can revisit if it causes some probrem.

Reply:
ya, was copying from quipucords for consistency. works fine in Mac too without, so I'll remove it (since that's the direction anyways).